### PR TITLE
GSoC'23 Ochi blog post

### PR DESCRIPTION
**Pull request by dkumiszhan**

### CODE REVIEW

**Comment by dkumiszhan** [comment]
@glaslos please take a look at this PR.

File: content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "GSoC 2023 Project summary: Ochi"
+authors: ["Lucas Rist"]
```
Comment (glaslos) [inline content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md:3]: Put your own name here :)

File: content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+date: "2023-08-24T11:00:06+01:00"
+# coverImage: "intel_owl_positive_reduced.png"
```
Comment (glaslos) [inline content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md:5]: Remove this line if you don't use it.

File: content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+tags: ["gsoc", "ochi"]
+---
+
+
+Our [GSoC](https://summerofcode.withgoogle.com/) student [Kumiszhan Dybyspayeva](https://github.com/dkumiszhan) was working for three months under the supervision of Lucas Rist on [Ochi Project](https://github.com/honeynet/ochi/), specifically on introducing new features such as event filtering using DSL, persisting queries and sharing an event.
```
Comment (glaslos) [inline content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md:10]:
```suggestion
Our [GSoC](https://summerofcode.withgoogle.com/) student [Kumiszhan Dybyspayeva](https://github.com/dkumiszhan) was working for three months under the supervision of Lukas Rist on [Ochi](https://github.com/honeynet/ochi/), specifically on introducing new features such as event filtering using a DSL, persisting filters and sharing an event. 
```

File: content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+Our [GSoC](https://summerofcode.withgoogle.com/) student [Kumiszhan Dybyspayeva](https://github.com/dkumiszhan) was working for three months under the supervision of Lucas Rist on [Ochi Project](https://github.com/honeynet/ochi/), specifically on introducing new features such as event filtering using DSL, persisting queries and sharing an event. 
+
+Read on for an overview of their achievements and how they successfully contributed towards Ochi and some considerations for the future.
+
+<!--more-->
+
+**Student**: Kumiszhan Dybyspayeva ([dkumiszhan](https://github.com/dkumiszhan))
+
+**Mentor**: Lucas Rist
```
Comment (glaslos) [inline content/blog/2023/08/28/gsoc-2023-project-summary-ochi/index.md:18]:
```suggestion
**Mentor**: Lukas Rist
```

**Review by glaslos — CHANGES_REQUESTED** [review]
Very minor suggestions only

**Review by glaslos — APPROVED** [review]

**Pull request merged**